### ISSUE TITLE
Multithread ticks of attached GameSystems

### ISSUE THREAD

**Issue by kkorolyov** [kkorolyov/pancake#41]
Systems associated with one another by their `Signature`s
Create graph of signature facets connected to systems
All disconnected subgraphs in this can be executed in parallel

**Comment by kkorolyov** [comment]
superseded by split into freeform `Pipeline`s
